docs/source/specs/typespec/main.tsp

```diff
@@ -215,6 +220,9 @@ namespace Workspaces {
     @get op list(
         @query limit?: int64 = 10;
         @query offset?: int64 = 0;
+
+        @doc("Defaults to all when param is not supplied.")
+        @query type?: WorkspaceTypesQueryParam;
```

Comment:
What do you think of adding the explicit default like this?
```suggestion
        @query type?: WorkspaceTypesQueryParam = WorkspaceTypesQueryParam.all;
```

Reply:
Well...I actually _tried_ to add this as a default the other day but must have been doing it wrong, because it wasn't compiling and I thought it was because of my custom enum. This works though, so updating! 😄